cmd/cmd.go

```diff
@@ -227,6 +228,18 @@ func (o *options) sternConfig() (*stern.Config, error) {
 			t = "{{.Message}}"
 		case "json":
 			t = "{{json .}}"
+		case "extjson":
+			t = "\"pod\": \"{{color .PodColor .PodName}}\", \"container\": \"{{color .ContainerColor .ContainerName}}\", \"msg\": {{extjson .Message}}"
+			if o.allNamespaces {
+				t = fmt.Sprintf("\"namespace\": \"{{color .PodColor .Namespace}}\", %s", t)
+			}
+			t = fmt.Sprintf("{%s}", t)
+		case "ppextjson":
+			t = "  \"pod\": \"{{color .PodColor .PodName}}\",\n  \"container\": \"{{color .ContainerColor .ContainerName}}\",\n  \"msg\": {{extjson .Message}}"
```

Comment:
I wanted to use `json.Marshal`, but found that using it would reorder the keys and make it difficult to see.